utils/clean-python.sh

```diff
@@ -14,6 +14,7 @@ rm -rf MANIFEST
 
 rm -rf python/jupyter/labextension
 rm -rf python/jupyter/nbextension
+rm -rf python/chemiscope/
```

Comment:
Why? This will remove the code! This file is intended to cleanup generated files only

```suggestion
```